eng/testing/ildescriptors/ILLink.Descriptor.Castle.xml

```diff
@@ -0,0 +1,13 @@
+<linker>
```

Comment:
All these files should have `Descriptors` in the name following the naming pattern set by the linker.

See the docs here:
https://github.com/mono/linker/blob/55bd0ebd00ddbf2cd817314b1cf6824003cfb463/docs/data-formats.md#descriptor-format